src/test/java/uk/ac/ebi/eva/pipeline/jobs/steps/AnnotationLoaderStepTest.java

```diff
@@ -73,21 +71,35 @@
 
     @Autowired
     private JobLauncherTestUtils jobLauncherTestUtils;
-    @Autowired
-    private JobOptions jobOptions;
 
     @Test
     public void shouldLoadAllAnnotations() throws Exception {
-        setUp();
-
-        JobExecution jobExecution = jobLauncherTestUtils.launchStep(BeanNames.LOAD_VEP_ANNOTATION_STEP);
+        String annotationFolder = temporaryFolderRule.getRoot().getAbsolutePath();
+        String dbName = mongoRule.getRandomTemporaryDatabaseName();
+        String collectionVariantsName = "variants";
+        String studyId = "1";
+        String fileId = "1";
+        String vepOutput = VepUtils.resolveVepOutput(annotationFolder, studyId, fileId);
+        String vepOutputName = Paths.get(vepOutput).getFileName().toString();
+        temporaryFolderRule.newGzipFile(VepOutputContent.vepOutputContent, vepOutputName);
+
+        mongoRule.restoreDump(getResourceUrl(MONGO_DUMP), dbName);
+
+        JobParameters jobParameters = new EvaJobParameterBuilder()
```

Comment:
The input study ID and the input VCF ID shouldn't be necessary for the annotation loader, see [AnnotationLoaderStepParametersValidator](https://github.com/EBIvariation/eva-pipeline/blob/develop/src/main/java/uk/ac/ebi/eva/pipeline/parameters/validation/step/AnnotationLoaderStepParametersValidator.java). They are for the VEP input generation step.